sycl/unittests/helpers/PiMockPlugin.hpp

```diff
@@ -1373,6 +1373,24 @@ inline pi_result mock_piextCommandBufferMemBufferCopyRect(
   return PI_SUCCESS;
 }
 
+inline pi_result mock_piextCommandBufferMembufferFill(
```

Comment:
I think our `operator==` for `node_impl` might need changes for the new CG Types. Could we have a unittest for fill and memset nodes to stress this?